### PR TITLE
Use CREATE_CUSTOM_NETWORK for all scalability jobs

### PR DESCRIPTION
**Pull request by wojtek-t**
@shyamjvs - I think in the past you fixed all the problems with that - is that true?
Is this supposed to work also in case of presubmits?

/hold

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml [hunk 1]
```diff
@@ -115,11 +117,3 @@ presets:
   # Increase size of default subnetworks to enable large clusters.
   - name: ENABLE_BIG_CLUSTER_SUBNETS
     value: "true"
-# Create a project k8s-jenkins-scalability-head and move this test there
-- labels:
-    preset-e2e-gci-gce-scalability: "true"
```
Comment (shyamjvs) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml]: There is one more place, where this preset is used - config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

Can you remove there also?

**Comment by shyamjvs** [comment]
> I think in the past you fixed all the problems with that - is that true?

Yeah. At least there are no issues I'm aware of.

> Is this supposed to work also in case of presubmits?

We're already using it for blocking presubmits.

**Comment by shyamjvs** [comment]
/lgtm
/hold

Feel free to merge once you fix the comment.

**Comment by shyamjvs** [comment]
Hmm.. I thought lgtm adds approve too.

/approve
/hold cancel

**Comment by wojtek-t** [comment]
> Hmm.. I thought lgtm adds approve too.

@shyamjvs - it used to in the past. But we switched to explicit approve a month or two ago.

**Comment by wojtek-t** [comment]
anyway - thanks!

**Comment by wojtek-t** [comment]
Also please lgtm if you can :-)

**Pull request merged**